server/portal/apps/workspace/api/views.py

```diff
@@ -81,6 +81,31 @@ def _get_app(app_id, user):
     return data
 
 
+# TODOV3: For retaining job data during v3 transition
+def _get_job_detail(client, job_id, username):
+    data = client.jobs.get(jobId=job_id)
+    q = {"associationIds": job_id}
+    job_meta = client.meta.listMetadata(q=json.dumps(q))
+    data['_embedded'] = {"metadata": job_meta}
+
+    # TODO: Decouple this from front end somehow
+    archiveSystem = data.get('archiveSystem', None)
+    if archiveSystem:
+        archive_system_path = '{}/{}'.format(archiveSystem, data['archivePath'])
+        data['archiveUrl'] = '/workbench/data-depot/'
+        data['archiveUrl'] += 'agave/{}/'.format(archive_system_path.strip('/'))
+
+        jupyter_url = get_jupyter_url(
+            archiveSystem,
+            "/" + data['archivePath'],
+            username,
+            is_dir=True
+        )
+        if jupyter_url:
+            data['jupyterUrl'] = jupyter_url
+    return data
```

Comment:
Looking at this one more time. Would the frontend historical view need any of this extra data, other than just the data returned from the initial tapis call?

Reply:
From what I could tell, I don't think so, but curious of your thoughts as well

Reply:
@rstijerina 🤔 If I'm not mistaken, it seems like none of the data (`archiveUrl`, `jupyterUrl`, `_embedded`) are used in the v2 frontend as well (other than in the fixtures). I preserved it because I thought it should be consistent with `state.jobDetails`. 
Should I delete the rest of the content?

Reply:
I think you're right about that. In v3, we are removing all this extra stuff already, so here, to simplify the v2 -> v3 diff, it might be simplest to leave it as is (reverting the `_get_job_detail` function), and then just storing the job detail in the new `data` field

Reply:
Gotcha!